mm2src/coins/hd_wallet/mod.rs

```diff
@@ -0,0 +1,515 @@
+use async_trait::async_trait;
```

Comment:
I wonder why mod.rs was added instead hs_wallet.rs?
I can see the general trend now is to avoid mod.rs files to have more descriptive file names

Reply:
I plan to make the `hd_wallet` directory a crate where I rename `mod.rs` to `lib.rs`. To make it a crate I need to remove any coin specific logic or structs from it, only `UtxoAddressFormat` is what's left then I can make it a separate crate.